Code/RDGeneral/types.cpp

```diff
@@ -28,6 +28,7 @@ const std::string NullBond = "NullBond";
 const std::string _2DConf = "_2DConf";
 const std::string _3DConf = "_3DConf";
 const std::string _AtomID = "_AtomID";
+const std::string _AtomRank = "_AtomRank";
```

Comment:
Should this be _CanonicalAtomRank for clarity?

Reply:
OK with me